### ISSUE TITLE
ForkTsCheckerWebpackPlugin v.6.0.2 watching mode loop

### ISSUE THREAD

**Issue by naturalfreak** [TypeStrong/fork-ts-checker-webpack-plugin#536]
os - Ubuntu 20.04
fork-ts-checker-webpack-plugin - 6.0.2
typescript - 4.0.5
webpack - 5.4.0

Using webpack in watch mode with combination with ForkTsCheckerWebpackPlugin and CopyPlugin,
watched files by CopyPlugin are not recompiled after change.
[Here small part of project files](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/files/5545187/forkplugin.zip), 
to run project:
npm install
npm run start:dev - to start webpack in watch mode
if you change file languages/lang.json, managed by CopyPlugin, file is not recompile.
With fork-ts-checker-webpack-plugin - 5.2.1 everything works as expected

**Comment by naturalfreak** [comment]
Fixed with copy-webpack-plugin@6.3.2

**Comment by brentvollebregt** [comment]
For anyone else that find this, I used the internal logging to identify what files were being changed to identify what was triggering the loop:

```typescript
new ForkTsCheckerWebpackPlugin({
  logger: {
    infrastructure: 'console'
  }
})
```

For me, the output js files (built files) were included in my tsconfig - thus any change would update these and then trigger a rebuild creating a loop. To fix this, I excluded the build directory from my tsconfig.
